### DISCUSSION TITLE
Duplicates Topics Confirmation

### DISCUSSION THREAD

**Discussion by AayushSaini101** [apache/rocketmq#7097] [Ideas]
Does RocketMQ allow to create of duplicate topics? I guess not. Whenever we create topics of the same name we get a message that is given below.
`create topic to 192.168.1.8:10911 success.`
<img width="1106" alt="image" src="https://github.com/apache/rocketmq/assets/60972989/ca5db133-6682-4af3-85ca-0e44d199390f">

* Can we use different messages like the topic is already present and cannot create the same name? or a confirmation of whether to create or topic or not? It would help to protect the older topic.

If it is possible I am willing to work on this issue and create the same.

**Comment by ShadowySpirits** [comment]
At present, the MQAdmin cli tools only offer the updateTopic command. However, we could improve this by dividing it into two separate commands: createTopic and updateTopic. The former allow users to create a new topic, while the latter only make changes to an already existing topic.
